packages/common/src/TransactionUtils.ts

```diff
@@ -318,6 +318,7 @@ export async function findBlockNumberAtTimestamp(
  * catch reverts that could not be seen at submission time.
  */
 export async function processTransactionPromiseBatch(transactions: Array<ExecutedTransaction>, logger: winston.Logger) {
+  if (transactions.length == 0) return;
```

Comment:
we produced logs in all cases this was called, even if there is no transactions to waint on.

Reply:
Is it overkill to log that there are 0 transactions before returning?